misk-hibernate/src/main/kotlin/misk/hibernate/SessionFactoryService.kt

```diff
@@ -192,6 +192,7 @@ internal class SessionFactoryService(
       "char" -> Char::class
       "float" -> Float::class
       "double" -> Double::class
+      "materialized_clob" -> String::class
```

Comment:
what if the lob does not have a character type?

Reply:
Good point, this can't be done here.